docs/UserGuide.md

```diff
@@ -257,10 +260,8 @@ Find by role example:
 
 Find by remark example:
 * `find r/` returns all persons who have an empty remark.
-* `find r/has a dog` returns all persons who have the remark `has a dog`.
-* `find r/dog` returns all persons who have the remark `has a dog`.
-* `find r/a has` returns all persons who have the remark `has a dog`.
-
+* `find r/has a dog` returns all persons who have the remark containing words `has a dog`.
```

Comment:
![image](https://github.com/AY2324S2-CS2103T-F12-1/tp/assets/110763159/b69e1ed0-0015-45af-ac8b-22c075b0f62e)

According to this, shall we change this `find r/has a dog returns all persons who have the remark containing words has a dog.` into `find r/has a dog returns all persons who have the remark containing all these words ['has', 'a', 'dog']`